apps/web/src/components/ParticipantWindow/ParticipantWindow.tsx

```diff
@@ -36,7 +35,6 @@ const useStyles = makeStyles((theme: Theme) =>
 export default function ParticipantWindow() {
   const classes = useStyles();
   const { appState } = useAppState();
-  const raisedHands = useRaisedHandsMap();
   const { room } = useVideoContext();
   const enqueueSnackbar = useEnqueueSnackbar();
 
```

Comment:
With this update, I'm not sure it makes sense anymore to have the `handleInvite` function live here. It's just passed to the ViewersList. Maybe it should live in the ViewersList component?